content/en/docs/concepts/scheduling-eviction/pod-eviction.md

```diff
@@ -0,0 +1,375 @@
+---
+title: Pod Eviction
```

Comment:
Is this for the eviction api (the one used when draining a node), or out-of-resource eviction?

Reply:
This seems to mix details of both.

Reply:
It is also somewhat strange to have kubelet resource management under a scheduling-related section.

Reply:
This is meant to include all situations where eviction happens (so from what I understand, kubelet, draining, the eviction API?). The section is Scheduling and Eviction so I put it there - what's your recommendation?

Reply:
I'm not entirely sure what scheduling and eviction have in common... They seem like two entirely separate topics.

Reply:
Hmm. What about under Concepts > Cluster Administration instead?

Reply:
That seems like a reasonable place

Reply:
> I'm not entirely sure what scheduling and eviction have in common... They seem like two entirely separate topics.

Eviction quite often triggers scheduling (of the replacement). I'm in favor of presenting the two concepts alongside each other; in fact, I'd like to include preemption in the mix as well. That way we can contrast preemption with eviction and explain how the two differ - easier if the concepts are presented as siblings.

Also, people who aren't cluster administrators are likely to want to learn about eviction, especially the kind where the kubelet triggers it without prompting.

Reply:
+1 i would move this out of scheduling section and make it clear this is specifically for local kubelet eviction / out-of-resource eviction

Reply:
> I'm not entirely sure what scheduling and eviction have in common... They seem like two entirely separate topics.

Eviction quite often triggers scheduling (of the replacement). I'm in favor of presenting the two concepts alongside each other; in fact, I'd like to include preemption in the mix as well. That way we can contrast preemption with eviction and explain how the two differ - easier if the concepts are presented as siblings.

Also, people who aren't cluster administrators are likely to want to learn about eviction, especially the kind where the kubelet triggers it without prompting.

Reply:
I'm still a little finicky with github so I'll finish up edits to content before I move it 😝

Reply:
@sftim makes a good point as well - it made more sense to me under the scheduling section. I will hold off on moving the topic until we decide where it needs a home

Reply:
We had a long discussion in #19081

After seeing this PR, 1/3 types of eviction directly relate to scheduling (currently the PR is not talking about preemption). The others are more about node health, which is under the kubelet domain.

Maybe a better home would be somewhere in https://kubernetes.io/docs/concepts/architecture/

But I know @sftim feels very strongly about this belonging within scheduling. So maybe it's is fine, as long as we change the URL from `scheduling-eviction` to `scheduling-and-eviction`. I thought we had done this already.

Reply:
> change the URL from `scheduling-eviction` to `scheduling-and-eviction`

We usually omit short words like “and” from URLs - for example, “Services And Networking” is represented in the URL as `services-networking`.